app/docs/0.11.x/getting-started/quickstart.md

```diff
@@ -20,7 +20,8 @@ supports PostgreSQL and Cassandra).
     Issue the following command to [start][CLI] Kong:
 
     ```bash
```

Comment:
I'm inclined to agree with @kikito here. The throwaway comment "(only first time)" is fairly ambiguous, giving no context to its use or value. We should be more explicitly about what's going on; at the least let's detail it a bit more elsewhere in the docs and link to it from these quickstart guides.